docs/laborok/02-android-ui-compose/index.md

```diff
@@ -1,8 +1,8 @@
 # Labor 02 Extra - Jetpackkel az égbe ☁ 🚀 ✨
 
-***Szerző: Püspök-Kiss Balázs***
+**_Szerző: Püspök-Kiss Balázs_**
```

Comment:
Nem hiszem, hogy kellene módosítani bármit is a Markdown fájl formázása miatt, de ha van érved, akkor shoot.

Reply:
A prettier végezte a dolgát a `ctrl+s`-re igazából

Reply:
IMO olvashatóbb egy fokkal hogy bold (**) és italics (_) ez szerintem ízlés kérdése